documentation/proc-pages/physics-models/fusion_reactions/plasma_bosch_hale.md

```diff
@@ -0,0 +1,92 @@
+# Bosch-Hale Methods
+
+These methods are still kept in `physics_functions.py` but outside the [`FusionReactionRate`](plasma_reactions.md) class.
+
+## Bosch-Hale Constants | `BoschHaleConstants`
+
+The `BoschHaleConstants` class is a data structure designed to hold the constants required for the Bosch-Hale calculation for a given fusion reaction.
+
+### Attributes
+
+- **bg (float)**: Represents the Gamow energy parameter.
+- **mrc2 (float)**: Represents the reduced mass energy term.
+- **cc1 (float)**: Coefficient for the first term in the Bosch-Hale polynomial.
+- **cc2 (float)**: Coefficient for the second term in the Bosch-Hale polynomial.
+- **cc3 (float)**: Coefficient for the third term in the Bosch-Hale polynomial.
+- **cc4 (float)**: Coefficient for the fourth term in the Bosch-Hale polynomial.
+- **cc5 (float)**: Coefficient for the fifth term in the Bosch-Hale polynomial.
+- **cc6 (float)**: Coefficient for the sixth term in the Bosch-Hale polynomial.
+- **cc7 (float)**: Coefficient for the seventh term in the Bosch-Hale polynomial.
+
+--------------------------
+
+## Volumetric Fusion Rate | `bosch_hale_reactivity()`
+
+This function calcualtes the relative velocity fusion reactivity $\langle \sigma v \rangle$ for each point in the plasma profile based on the temperature.
+
+ |  Input Variable             |    |
+    |----------------------------------|-----------|
+    | Array of temperature values for the plasma profile            | `temperature_profile`  |
+    | Bosch-Hale constants for the specific reaction                   | `reaction_constants`  |
+
+$$
+\theta = \frac{\text{T}}{\left[1-\frac{\text{T(C2+T(C4+TC6))}}{1+\text{T(C3+T(C5+TC7))}}  \right]}
+$$
+
+$$
+\xi = \left(\frac{\text{B}_\text{G}^2}{4\theta}\right)^{\frac{1}{3}}
+$$
+
+$$
+\langle \sigma v \rangle = \text{C1} \times \theta \times \sqrt{\frac{\xi}{m_{\text{r}}\text{c}^2\text{T}^3}} \times e^{-3\xi}
+$$
+
+This will output a numpy.array for of the relative velocity fusion reactivity $\langle \sigma v \rangle$ for each point in the temperature profile in units of $[\text{m}^3\text{s}^{-1}]$ After calculation each value is multiplied by $10^{-6}$ as the original Bosch-Hale calculation[^1] give the output in $[\text{cm}^3\text{s}^{-1}]$
+
+--------------------------------
+
+## Fusion Rate Integral | `fusion_rate_integral()`
+
+|  Input Variable             |    |
```

Comment:
add "Variable Name" to second table heading?

Reply:
Added